lib/cli.js

```diff
@@ -18,8 +18,7 @@
 const fs = require("fs"),
     path = require("path"),
     { promisify } = require("util"),
-    { ESLint } = require("./eslint"),
-    { FlatESLint } = require("./eslint/flat-eslint"),
+    { ESLint, FlatESLint } = require("./eslint"),
```

Comment:
Based on the previous change I'd expect you really meant `require("./api")` here?

Reply:
Yes, of course 🤦